app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-view.hbs

```diff
@@ -153,7 +153,22 @@
       </div>
     </Auk::Panel::Body>
   {{/if}}
-  {{#if @agendaitem.privateComment}}
+  {{#if (and (user-may "manage-agendaitems") @subcase.internalReview)}}
+   <Auk::Panel::Body class="auk-u-maximize-height auk-u-bg-alt au-u-padding-top-small au-u-padding-bottom-small">
+      <div class="au-u-flex au-u-flex--spaced-tiny au-u-flex--vertical-start">
+        <div class="au-u-flex-item--fixed au-u-flex au-u-flex--spaced-tiny au-u-flex--vertical-center auk-u-text-bold">
+          <AuIcon @icon="lock-closed" />
+          <span>{{t "private-comment-title"}}:</span>
+        </div>
+        <pre
+          class="au-u-flex-item--fill auk-u-text-pre-line auk-u-text-muted"
+          data-test-agendaitem-titles-private-comment
+        >
+          {{@subcase.internalReview.privateComment}}
+        </pre>
+      </div>
+    </Auk::Panel::Body>
```

Comment:
note that I kept the styling for agendaitem to be the same.
The views in subcase and submissions are different ('interne opmerking" is above, not on the side.)
Didn't implement this change here.
Not sure what looks better.